packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -92,6 +92,26 @@ class SlidePanel extends React.Component {
       customProps.className,
     ]);
 
+    const content = (panelPosition === 'start') ? (
```

Comment:
Is there an `enum`-like object for 'start' and 'end' that could be leveraged here?

Reply:
Added an enum object and made the necessary changes to incorporate it in the examples as well.